tests/integration_tests/test_sql_task.py

```diff
@@ -466,6 +471,7 @@ def test_filter_on_data_categories(
     rule_target.delete(db)
 
 
+@pytest.mark.integration_postgres
```

Comment:
Similarly, TestRetrievingData below could be marked with integration at the class level, and also marked with just `postgres`